index.html

```diff
@@ -0,0 +1,177 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio ErnestKorir</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700;800&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header class="toolbar">
+        <a href="#" class="logo">My Logo</a>
+        <nav class="fas fa-bars"></nav>
+    </header>
+    <section class="home-page-section">
+        <h1 class="Right Intro">Namaste!<br> I'm Ernest</h1>
+        <p class="description">I’m a software developer! I can help you build a product , feature or website. Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.</p>
+
+        <div class="Right contact">
+            <a class="media-text">LET’S CONNECT</a>
+
+            <ul class="right-icons">
+                <li><a href="https://twitter.com/Ernestbore3" target="_blank" class="media-link" rel="noopener"><i
+                            class="fab fa-twitter"></i></a></li>
+                <li><a href="https://www.linkedin.com/in/ernest-bore-3b6042211/" target="_blank" class="media-link"
+                        rel="noopener"><i class="fab fa-linkedin"></i></a></li>
+                <li><a href="https://medium.com/" target="_blank" class="media-link" rel="noopener"><i
+                            class="fab fa-medium-m"></i></a></li>
+                <li><a href="https://github.com/ernestkorir" target="_blank" class="media-link" rel="noopener"><i
+                            class="fab fa-github"></i></a></li>
+                <li><a href="https://angel.co/ernestkorir" target="_blank" class="media-link" rel="noopener"><i
+                            class="fab fa-angellist"></i></a></li>
+            </ul>
+        </div>
+    </section>
+    <section class="work">
+        <div class="card-work-1">
+            <div class="work-image"></div>
+            <div class="left-block">
+                <p class="Tonic">Tonic</p>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
```

Comment:
- [ ] **_all of your buttons_** should follow the button styling: Active, Hover and Pressed like shown in figma , please fix it 🙏 

![image](https://user-images.githubusercontent.com/89282221/192088431-cffd1efb-6762-4778-aed2-d7f8d8781c21.png)